.github/workflows/integration-tests.yml

```diff
@@ -76,6 +76,8 @@ jobs:
 
   aws-nuke-integration-tests:
     name: AWS Nuke Integration Tests
+    if: always() && needs.aws-deploy-integration-tests.result == 'success'
+    continue-on-error: true
     needs:
       - aws-load-tests
```

Comment:
This rule has a higher priority than the if above right?

Reply:
It just let him know that it has to run after the `aws-load-tests` job, but the conditional about plus the `continue-on-error` property set to true, makes it run even when the previous step fails if the deployment job succeeds.

Reply:
Wonderful 👍